DesignSafe-CI/portal
# task/WP-673:  Interactive Modal Redesign

## Overview: ##

Redesigns the Interactive Modal based on feedback. Refactored to use shared context.

## PR Status: ##


## Related Jira tickets: ##

* [WP-673](https://tacc-main.atlassian.net/browse/WP-673)

## Testing Steps: ##
1. Select an interactive app, like https://designsafe.dev/workspace/opensees-interactive?appVersion=3.7.0
2. Confirm submit button text is changed to "Launch Session"
3. Submit job, and confirm UI looks like photo 2 below
4. When session is ready, confirm UI looks like photo 3 below

## UI Photos:

1. New submit button for interactive apps
![Screenshot 2024-10-03 at 6 50 50 PM](https://github.com/user-attachments/assets/0c21a794-c505-4164-b683-537827da8ba1)


2. Upon interactive app launch
![Screenshot 2024-10-03 at 6 50 08 PM](https://github.com/user-attachments/assets/791679bf-fd30-4bb9-ac90-d57b14a72985)


3. When session is ready and modal is still open, or if user clicks "Open" button when session is ready
![Screenshot 2024-10-03 at 6 48 09 PM](https://github.com/user-attachments/assets/a57e5805-298e-44e5-a276-94d8638f0cd1)
